src/gui/owncloudgui.cpp

```diff
@@ -145,7 +148,7 @@ void ownCloudGui::slotOpenSettingsDialog()
     // if account is set up, start the configuration wizard.
     if (!AccountManager::instance()->accounts().isEmpty()) {
         if (_settingsDialog.isNull() || QApplication::activeWindow() != _settingsDialog) {
-            slotShowSettings();
+            slotOpenMainDialog();
```

Comment:
I'm confused, shouldn't that be `slotShowSettings()`?

I mean the check on the if is all about the `_settingsDialog` being active so it's surprising to then go and open something else.

Reply:
reverted.